workers/vulns.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+#
+# ronin-app - a local web app for Ronin.
+#
+# Copyright (C) 2023 Hal Brodigan (postmodern.mod3@gmail.com)
+#
+# ronin-app is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-app is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with ronin-app.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+require 'sidekiq'
+require 'ronin/vulns/vuln'
+
+module Workers
+  #
+  # A worker that updates all [ronin repos][ronin-repos].
+  #
+  # [ronin-repos]: https://github.com/ronin-rb/ronin-repos#readme
+  #
+  class Vulns
+
+    include Sidekiq::Worker
+    sidekiq_options queue: :vulns, retry: false, backtrace: true
+
+    Params = Dry::Schema::JSON() do
+      required(:url).filled(:string)
+    end
```

Comment:
Need specs for `Params`. You can copy/paste existing specs from `spec/workers/` which tests the `Params` of the worker class.